### PR TITLE
feat: Support to disable shared notes and captions via disabledFeatures

### PR DESCRIPTION
**Pull request by gustavotrott**
Moving on with the implementation of `disabledFeatures` param #14293.

Add new features to disableable list: `sharedNotes` and `captions`.

### CODE REVIEW

**Comment by pedrobmarin** [comment]
Hmmm, I believe this will break the closed captions

**Comment by pedrobmarin** [comment]
You should aim for the group creation instead of the user's session. Maybe move your test over here https://github.com/bigbluebutton/bigbluebutton/blob/506468e4b75427cc8e7b0487cc1911578058af28/akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/pads/PadCreateGroupReqMsgHdlr.scala#L24

At `Pads`, add a function to test if `disabledFeatures` contains `sharedNotes` and check if `msg.body.externalId` is `notes`. I believe that's the pad's externalId when creating the shared notes group.

**Comment by gustavotrott** [comment]
Thanks @pedrobmarin,

It was indeed breaking Closed Captions.
I just made your suggestions and now CC is working fine when sharedNotes is disabled.

File: akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/pads/PadCreateGroupReqMsgHdlr.scala [hunk 1]
```diff
@@ -21,7 +21,17 @@ trait PadCreateGroupReqMsgHdlr {
...
-    if (!Pads.hasGroup(liveMeeting.pads, msg.body.externalId)) {
+    val padEnabled = {
+      if (msg.body.model == "notes" && liveMeeting.props.meetingProp.disabledFeatures.contains("sharedNotes")) {
+        false
+      } else if (msg.body.model == "captions" && liveMeeting.props.meetingProp.disabledFeatures.contains("captions")) {
+        false
+      } else {
+        true
+      }
+    }
```
[the remark this answers is not in the document]
Comment (gustavotrott) [reply akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/pads/PadCreateGroupReqMsgHdlr.scala]: Yes, it worked well! Thanks for the suggestion

**Pull request merged**